community-operators/mongodb-enterprise-upstream/v0.3.2/mongodb.v0.3.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,311 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments:
+      - name: mongodb-enterprise-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              k8s-app: mongodb-enterprise-operator
+          template:
+            metadata:
+              labels:
+                k8s-app: mongodb-enterprise-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: MANAGED_SECURITY_CONTEXT
+                  value: "true"
+                - name: OPERATOR_ENV
+                  value: prod
+                - name: MONGODB_ENTERPRISE_DATABASE_IMAGE
+                  value: quay.io/mongodb/mongodb-enterprise-database
```

Comment:
Should this image be tagged?

Reply:
Images referenced in CSVs should always be tagged.